source/_components/switch.raspyrfm.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: page
+title: "RaspyRFM Switch"
+description: "Instructions on how to integrate RaspyRFM switches into Home Assistant."
+date: 2018-12-08
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: seegelsysteme.png
+ha_category: Switch
+ha_release: 0.85
+---
+
+The `raspyrfm` component adds support for cheap RC 433 MHz outlets via one of the supported gateways.
+
+Initially this component was created to support the Simple Solutions `ConnAir` gateway which has been discontinued. There are custom alternatives that reimplemented the protocol used by the ConnAir though like this [ConnAir emulator](https://github.com/Phunkafizer/RaspyRFM/blob/master/connair.py) wich can be used in conjunction with the [RaspyRFM-II](https://www.seegel-systeme.de/produkt/raspyrfm-ii) rc module for a Raspberry Pi which can be used.
```

Comment:
Nah, I was referring to the existing phrase that had a typo in "which".

Reply:
:man_facepalming: fixed it :laughing: